R/ds_search_sql.R

```diff
@@ -13,9 +13,10 @@
 #' }
 #' @importFrom httr status_code
 
-ds_search_sql <- function(sql, url = get_default_url(), as = 'list', ...) {
+ds_search_sql <- function(sql, url = get_default_url(), as = 'list',
+                          config = httr::add_headers(Authorization = get_default_key()), ...) {
```

Comment:
why not just use `key = get_default_key()`? in the function definition, and then `add_headers(Authorization = key)` internally in the `GET` call